packages/react/src/components/CommandCenter/Input.tsx

```diff
@@ -0,0 +1,52 @@
+import { useCallback, useEffect, useState, forwardRef } from 'react';
+import styles from './styles.module.css';
+
+type InputProps = {
+    value: string;
+    onChange: (value: string) => void;
+    placeholder: string;
+    label: string;
+};
+
+export const Input = forwardRef<HTMLInputElement, InputProps>(function Input({
```

Comment:
Simple input component. 
This component has some internal state as well since it's the kind of state we don't really need to access this state externally.